packages/utils/src/extension.js

```diff
@@ -54,6 +56,13 @@ export const alarms = {
   upsertAlarm,
 }
 
+export const useTranslation = () => ({
+  t: key => {
+    const convert = pascalize(key)
+    return chrome.i18n.getMessage(convert)
```

Comment:
We also here should probably default to cannot translate string since it returns undefined if not found.


<img width="854" alt="Screen Shot 2019-11-21 at 7 57 31 AM" src="https://user-images.githubusercontent.com/578259/69354235-a0925a00-0c34-11ea-93a1-a2977da1d094.png">[ref](https://developer.chrome.com/extensions/i18n)

Also now thinking about this we are also going to have to support the params in the second argument to t.

https://github.com/mujo-code/source/blob/master/packages/extension/src/components/settings/index.js#L78


eg.

```javascript
chrome.i18n.getMessage(convert, ...args) || 'No translation found'
```